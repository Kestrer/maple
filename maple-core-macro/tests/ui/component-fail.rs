#![allow(non_snake_case)]

use maple_core::prelude::*;

fn Component() -> HtmlElement {
    template! {
        div
    }
}

fn compile_fail() {
    template! { UnknownComponent() };

    template! { Component };
    template! { Component(1) };
}

fn main() {}
