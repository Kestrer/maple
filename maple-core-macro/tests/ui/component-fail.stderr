error: expected an assignment expression
  --> $DIR/component-fail.rs:15:27
   |
15 |     template! { Component(1) };
   |                           ^
